packages/template-retail-react-app/app/components/swatch-group/swatch.jsx

```diff
@@ -38,6 +38,7 @@ const Swatch = (props) => {
             }}
             aria-checked={selected}
             variant="outline"
+            role="radio"
```

Comment:
Apparently upgrading lighthouse causes our a11y score to drop, I fixed one of the a11y problem to get the score back to 88.